pontis-ui/src/components/Navigation/NavHeader.tsx

```diff
@@ -30,22 +31,28 @@ const resources: Resource[] = [
     href: "#",
     icon: CodeIcon,
   },
+  {
+    name: "Features",
+    description: "Learn about what makes Pontis special.",
+    href: "/features",
+    icon: PuzzleIcon,
+  },
   {
     name: "Publishers",
     description: "Meet our data publishers.",
     href: "/publishers",
     icon: SpeakerphoneIcon,
   },
-];
-
-// List of resources displayed in the more tab
-const additional = [
   {
     name: "Protocols",
     description: "Take a look at who is already using Pontis.",
     href: "/protocols",
     icon: ViewListIcon,
   },
+];
+
+// List of resources displayed in the more tab
+const additional = [
```

Comment:
Assuming that what is shown in the more tabs depends on screen size?

Reply:
Not exactly. On mobile it's all a burger button.